torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -5319,6 +5319,54 @@ def test_post_localSGD_optimizer_step_reload(self):
                     tmp_file
                 )
 
+        @sandcastle_skip_if(
+            BACKEND not in DistTestCases.backend_feature["ddp"],
+            f"The {BACKEND} backend does not support DistributedDataParallel"
+        )
+        @skip_if_no_gpu
+        def test_DistributedDataParallel_SyncBatchNorm_Channels_Last_3D(self):
+            group, group_id, rank = self._init_global_test()
+            num_processes = dist.get_world_size()
+            local_bs = 2
+            bs_offset = int(rank * 2)
+            global_bs = int(num_processes * 2)
+
+            model = ONLY_SBN_NET
+            model_gpu = copy.deepcopy(model).cuda(rank)
+            model_DDP = nn.parallel.DistributedDataParallel(
+                model_gpu, device_ids=[rank]
+            )
+
+            memory_format = torch.channels_last_3d
```

Comment:
looks like preparing inputs is the only difference with channels last 2d, can you refactor everything else into a helper function for these 2 tests?